solr/solr-ref-guide/src/authentication-and-authorization-plugins.adoc

```diff
@@ -118,20 +121,13 @@ An authentication plugin consists of two parts:
 * All of the content in the authentication block of `security.json` would be passed on as a map to the plugin during initialization.
 * An authentication plugin can also be used with a standalone Solr instance by passing in `-DauthenticationPlugin=<plugin class name>` during startup.
 
-=== Available Authentication Plugins
-
-Solr has the following implementations of authentication plugins:
-
-* <<kerberos-authentication-plugin.adoc#kerberos-authentication-plugin,Kerberos Authentication Plugin>>
-* <<basic-authentication-plugin.adoc#basic-authentication-plugin,Basic Authentication Plugin>>
-* <<hadoop-authentication-plugin.adoc#hadoop-authentication-plugin,Hadoop Authentication Plugin>>
-* <<jwt-authentication-plugin.adoc#jwt-authentication-plugin,JWT Authentication Plugin>>
+See <<securing-solr.adoc#securing-solr,Securing Solr>> for a list of available authentication plugins.
```

Comment:
I'm not sure about this change (and note you did it for the authorization section too, so this comment applies there too). The pages for the plugins are sub-pages of this one, so navigation-wise it makes more sense to bring people deeper into the hierarchy from here instead of bouncing them one level up just so they can go a level back down.

If the change was made to make it easier to maintain the list and not have to repeat it across pages, one option would be to insert a tag around the original list, and then define an include directive here to pull the tagged section from the other page to this one. That allows maintenance in a single place but inclusion wherever it's relevant.

The syntax is explained here: https://asciidoctor.org/docs/user-manual/#by-tagged-regions. Insert tags like this:

```
// tag::<tag-name>[]
content you want to be tagged
// end::<tag-name>[]
```
Wherever you want to put the tagged content, you use a syntax like this:

```
include::<filename.adoc>[tag=<tag-name>]
```